index.tsx

```diff
@@ -6,7 +6,7 @@ const isIos = Platform.OS === "ios";
 
 type Props = { autoHidden: boolean };
 
-export const HomeIndicator = (props: Props) => {
+export const HomeIndicator: React.FC<Props> = (props) => {
```

Comment:
the problem with React.FC is that it implicitly adds children to the props. 
`export const HomeIndicator = (props: Props): ReactElement | null => {` could work, but this is a bit unusual

Reply:
That is a good point and a reason to move away from using `React.FC` in general. Your alternative seems reasonable to me, and for TypeScript support helps us more than `any`, why would it be unusual in your opinion?